### DISCUSSION TITLE
Have the slider border follow the combo color

### DISCUSSION THREAD

**Discussion by Rekunan** [ppy/osu#16056] [Ideas]
(original idea from https://youtu.be/BhvohWDZCq4?t=439)

It'd be similar to how slider borders follow the combo color in traceable.

I'd be fine if it was just like an option in the skin.ini file.

**Comment by peppy** [comment]
We don't intend on adding new settings to the `skin.ini` file, but it's something we can consider once skin configuration is in a good state.
